lib/board.rb

```diff
@@ -0,0 +1,50 @@
+class Board #represents the data and logic of a Tic-tac-toe game board
+  require 'pry'
+  attr_accessor :cells
+  #property cells that stores the data of the state of the board in an array
+  @cells = []
+
+  def initialize
+    @cells = Array.new(9, " ") #or you can call reset! method
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.